lib/fs.js

```diff
@@ -293,7 +303,12 @@ ReadFileContext.prototype.close = function(err) {
   req.oncomplete = readFileAfterClose;
   req.context = this;
   this.err = err;
-  binding.close(this.fd, req);
+
+  if (this.isUserFd) {
+    req.oncomplete.call(req, null);
```

Comment:
It should be in its own paragraph and prefixed with `Note:`. No need for that to be bold. But having that is common in documentation.

Thanks for working w/ me on this.